app.js

```diff
@@ -28,6 +28,9 @@ app.locals.title = `${capitalized(projectName)}- Generated with Ironlauncher`;
 const index = require('./routes/index');
 app.use('/', index);
 
```

Comment:
Here you need to import the session configuration that you created on /config/session.config.js